### PR TITLE
JTD-027 Created the project delivery employee table

### PR DESCRIPTION
**Pull request by dylanmckk**

### CODE REVIEW

File: src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CREATE TABLE projectDeliveryEmployees (
+    projectId int NOT NULL AUTO_INCREMENT,
+    deliveryEmployeeId int NOT NULL AUTO_INCREMENT,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql]: why is this auto increment?

**Review by dana-kainos — CHANGES_REQUESTED** [review]
finish date can be null in case its not specified in advance

change TIMESTAMP to DEFAULT CURRENT_DATE() or similar - please test before you push

File: src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CREATE TABLE projectDeliveryEmployees (
+    projectId int NOT NULL AUTO_INCREMENT,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql]: why is this auto incrememnt

File: src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    deliveryEmployeeId int NOT NULL AUTO_INCREMENT,
+    startDate DateTime NOT NULL TIMESTAMP,
+    finishDate DateTime NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V8__create_project_delivery_employee_table.sql]: should this be in this table? start date and finish date is unique to the project unless its an employee start date in which case it has an incorrect name, Also why timestamp?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by dana-kainos — APPROVED** [review]

**Review by lukemKainos — APPROVED** [review]

**Pull request merged**